controllers/product_product.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+
+from odoo import http, _
+from odoo.http import request
+from odoo.exceptions import ValidationError
+
+from .main import UdesApi
+
+class Product(UdesApi):
+
+    @http.route('/api/product-product/', type='json', methods=['GET'], auth='user')
+    def get_product(self, id=None, query=None, fields_to_fetch=None):
```

Comment:
Can we change `id` to `identifier`, since `id` is a python core function that we are shadowing? It's bad practice to shadow the core functions.

Reply:
sure, I'll change it to product_id.